tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -33,6 +33,11 @@ internal static class Resources
         public const string AspNetCoreProcessNotFound = "Could not find the ASP.NET Core applicative process.";
         public const string VersionConflict = "Tracer version 1.x can't be loaded simultaneously with other versions and will produce orphaned traces. Make sure to synchronize the Datadog.Trace NuGet version with the installed automatic instrumentation package version.";
 
+        public const string TracingWithBundleProfilerPath = "Assuming process is traced with Datadog.Trace.Bundle Nuget, configuration(CORECLR_PROFILER_PATH or COR_PROFILER_PATH) was found for this service: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=nuget#install-the-tracer";
```

Comment:
on line 31, should we update to add the link in the ms doc? Like this one? https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1#create-the-iis-site:~:text=CLR%20version%20to-,No%20Managed%20Code,-%3A ?

Reply:
Not sure if just the plain https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1#create-the-iis-site but unless anyone objects I don't mind adding the full link, will do that!